src/Shared/ReuseableStringBuilder.cs

```diff
@@ -254,7 +254,7 @@ private static class ReuseableStringBuilderFactory
             /// because we could otherwise hold a huge builder indefinitely.
             /// This size seems reasonable for MSBuild uses (mostly expression expansion)
             /// </summary>
-            private const int MaxBuilderSize = 1024;
+            private const int MaxBuilderSize = 2048;
```

Comment:
Changing MaxBuilderSize is something we're currently experimenting with. Could you provide any data you may have gathered to show how much this reduces memory allocations?